.github/workflows/release.yml

```diff
@@ -26,6 +28,46 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
+      - name: Run Trivy static code analysis
```

Comment:
What is the target of this step here? Is this scanning every dockerfile in the repository? We should document it and add a way to exclude specific folders

Reply:
Yes, looks at every Dockerfile, also the deprecated ones.